### ISSUE TITLE
Accessibility: Governance/Decision_Making

### ISSUE THREAD

**Issue by Bhaybhiisparks** [jupyterhub/outreachy#117]
### What page is this for?

https://jupyter.org/governance/decision_making.html

### WAVE accessibility report

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1274440430

<img width="227" alt="ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ" src="https://user-images.githubusercontent.com/86384288/197408507-5c72b8b3-309e-4ce1-bb8f-d624f91f5a47.png">

<img width="226" alt="zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz" src="https://user-images.githubusercontent.com/86384288/197408529-2411ca7b-3022-43ff-af9c-087f3c1dce54.png">

### The HTML element to be changed

```html
1. Empty Form Labels: 
-- <label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

--<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

-- <label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>

-- <label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

-- <label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

2.  Multiple Form Labels: 
-- <input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">

-- <input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">

3. Invalid Document Language:
<html style="margin-left: 380px;">

4. Contrast Errors:
-- <span class="pre">
seeking
</span>

-- <span class="pre">
consensus
</span>

-- <span class="pre">
vote
</span>

-- <span class="pre">
decision
</span>

-- <span class="pre">
vote
</span>

-- <span class="pre">
made
</span>

5. Linked Image with alternative text contained in a link
<a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">
</a>
```

### Your proposed HTML (or CSS), after change

```html
1. Personally, I think the label tags should be removed. Most especially since they are not associated with any form element and they contain none text elements. The <label> tag defines a label for an input element, a select or textarea element. So, instead of using labels, we could enclose them in span tags or div tags, or enclose them in span tags enclosed in div tags and pass down the attributes (mostly data-placement from what I can see here) to the span/div tags. I will give examples of each starting from SPAN-DIV-SPAN ENCLOSED IN A DIV

-- <span class="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</span>

--<div class="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</div>

-- <div class="toctree-checkbox-3">
<span class="checkbox-3"> <i class="fas fa-chevron-down"></i> </span>
</div>

-- <div class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</div>

-- <label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

NOTE: I chose the class names assuming they are not already associated with another element.


2. I went through the code and could not find how the label was used for multiple form controls. From the code, each label is actually associated with just one form control. This could probably be an error from wave, or a control element somewhere in the page has a similar NAME or ID attribute value . Albeit if this issue was actually somehow present in the code, I would suggest detaching one of the form controls from the label in question by changing it's ID and NAME value and giving it its own label.
Multiple Form Labels: 
-- <input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">

-- <input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">

3. Add a language attribute

<html style="margin-left: 380px;" lang="en">

4. Add the style filter: contrast(..%) depending on the element's background color and position on the page or change the color of the text based on the background color. In this example, I changed the color of the text(made it darker).

<span class="pre" style="color: hsla(360, 100%, 50%, 1);">
made
</span>

5. Set alt attribute to ""
<a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="">
</a>
6.
```

**Comment by minrk** [comment]
Great suggestions, thank you!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
